python/CONTRIBUTING.md

```diff
@@ -72,7 +72,13 @@ Requires AWS S3 credentials.
 
 1. Publish shell script (which will use the Docker image tagged with the same version)
 
-   1. Version bucket
+   1. Verify that the version looks good:
+
+      ```bash
+      grep 'readonly VERSION=' create_service_diagnostics_bundle.sh
+      ```
```

Comment:
This was added when the plan was to `sed -i` the script. It's not needed if we don't do it.
OTOH the command below could benefit from reading the version from the script. But that's outside the scope of this PR.

Reply:
removed